version.py

```diff
@@ -44,4 +44,4 @@
 #   historical commits sharing the same major version, in which case a
 #   minor version bump suffices.
 
-PROVISION_VERSION = '81.2'
+PROVISION_VERSION = '81.3'
```

Comment:
This needed a major bump, because the new `semgrep` name does not work with the old code that invokes `sgrep-lint`.

Reply:
It doesn't, because you run `sgrep` with the older versions.  Only with your fix to delete the old binary do we need it.

Reply:
It does. Suppose your very first provision is at `81.3` which gives you `semgrep`, then you go back to `81.2` which expects `sgrep-lint`. Or suppose you provisioned at `75.5` or before with neither, then go forward to `81.3` and back to `81.2`.